tirocks-sys/crocksdb/c.cc

```diff
@@ -4092,88 +4032,60 @@ struct crocksdb_encryption_key_manager_impl_t : public KeyManager {
 
   Status GetFile(const std::string& fname,
                  FileEncryptionInfo* file_info) override {
-    crocksdb_file_encryption_info_t info;
-    info.rep = file_info;
     Status s;
-    get_file(state, fname.c_str(), &info, &s);
+    crocksdb_file_encryption_info_t info;
+    get_file(state, fname, &info, &s);
+    file_encryption_info_to_cpp(info, file_info);
```

Comment:
why can't use `KeyManager` directly?

Reply:
You mean why not use `FileEncryptionInfo` right? Because it contains std::string.

Reply:
Now `FileEncryptionInfo` is used directly by manually definition.